clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/AtomicOperations.java

```diff
@@ -38,6 +38,7 @@
   public static final String START_SERVER_GROUP = "startServerGroup";
   public static final String STOP_SERVER_GROUP = "stopServerGroup";
   public static final String SET_STATEFUL_DISK = "setStatefulDisk";
+  public static final String UPSERT_JOB_DISRUPTION_BUDGET = "upsertJobDisruptionBudget";
```

Comment:
Kubernetes has a "Pod Disruption Budget". I wonder if we should make this a little more abstract, e.g. `upsertDisruptionBudget`, to allow them to reuse this.

Reply:
Makes sense , updated the operation to `upsertDisruptionBudget`